api/src/Entity/DayResponsible.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/*
+ * Copyright (C) 2011 Urban Suppiger
+ *
+ * This file is part of eCamp.
+ *
+ * eCamp is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * eCamp is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with eCamp.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+#[ApiResource]
+class DayResponsible extends BaseEntity implements BelongsToCampInterface {
+    /**
+     * @ORM\ManyToOne(targetEntity="Day", inversedBy="dayResponsibles")
+     * @ORM\JoinColumn(nullable=false, onDelete="cascade")
+     */
+    public ?Day $day;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="CampCollaboration", inversedBy="dayResponsibles")
+     * @ORM\JoinColumn(nullable=false, onDelete="cascade")
+     *
+     * @var CampCollaboration[]
+     */
+    public ?CampCollaboration $campCollaboration;
+
+    public function getPeriod(): ?Period {
+        return $this->day->getPeriod();
+    }
```

Comment:
Methode ist auch im alten Backend unbenutzt, und soll nicht direkt hier in der API exposed werden -> Getter entfernen